webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -397,6 +407,9 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
         this.error = true;
         this.loading = false;
         console.error('Error loading contacts', err);
+      })
+      .finally(() => {
+        trackPerformance?.stop({ name: 'contact_list:query', recordApdex: true });
```

Comment:
This differs from the telemetry record `search:contacts:<filter[:filter]>`. 
The `contact_list:query` tracks the query in the Contacts Tab only and includes the whole query process to display the result, like formatting contacts, setting up the scroll etc. This one also tracks the Apdex. 
While `search:contacts:<filter[:filter]>` tracks the process from search service only, which can be called by other components (not just Contact list).